karma.conf.js

```diff
@@ -0,0 +1,69 @@
+// Karma configuration
+// Generated on Wed Dec 27 2017 10:17:29 GMT-0600 (CST)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine', 'browserify'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'assets/**/*.js'
+    ],
+
+
+    // list of files / patterns to exclude
+    exclude: [
+    ],
+
+
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+        'assets/**/*.js': ['browserify']
+    },
+
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['progress'],
```

Comment:
I think you will need some sort of coverage reporter from Coveralls.

Reply:
Yes you're right - I'll add that as part of WDFN-9